lamp/index.php

```diff
@@ -1,12 +1,23 @@
 <?php
-    $page_title = "Home";
-    $auth_needed = false;
+$page_title = "Home";
+$auth_needed = false;
+$center_page = true;
 ?>
 <?php require_once("lib/page-setup.php") ?>
 <?php include('common/header.php'); ?>
-Home page!
-<br />
-<a class="btn btn-primary" href="signin.php">Log In</a>
-<br />
-<a class="btn btn-primary" href="signup.php">Sign Up</a>
+<h2 class="homepageheader">Welcome to Burrow!</h2>
+<hr/>
+<h2 class="maintitleheader">How to get started:</h2>
+<br/>
+<p>To get started with Burrow, you need to create an account with the button below if you do not have one already.</p>
+<p>After you have created your account, join the classes that you are enrolled in this semester on the course search page.</p>
+<p>Once you have joined all your courses, you are ready to start chatting!</p>
+<hr/>
+<br/>
+<h2 class="maintitleheader">If you already have an account:</h2>
```

Comment:
Line 17 + 18 could be removed, just leave "sign-in" button, feels a bit redundant to have line 17 to me

Reply:
I agree (along with your next comment). It could make for a simpler UI; less text to read, more focus on the important actions (signing in or signing up).